docs/pages/corefunctionality/sources.rst

```diff
@@ -8,6 +8,7 @@ There are a few different potential sources for Simulations:
 .. code-block:: java
 
     SimulationSource.classpath("simulation.json"); //classpath
+    SimulationSource.classpathBelowHoverflyDir("simulation.json"); //classpath prefixed with /hoverfly
```

Comment:
Yes good idea - wasn't sure how to infer it's the classpath prefixed with `hoverfly/`